<?xml version="1.0"?>
<launch>
	<arg name="target_distance" default="0.25"/>
	<arg name="threshold_distance" default="0.40"/> 
	<arg name="Kp" default="0.0003"/>
	<arg name="Ti" default="0.005"/>	
	<arg name="Td" default="0.005"/>
	<arg name="line_v" default="0.2"/>	
	<arg name="angular_z" default="0.2"/>
	<arg name="num_f" default="180"/>
	<arg name="num_l" default="360"/>
	<arg name="num_r" default="0"/>
	<arg name="num_rf" default="30"/>
	<arg name="num_rh" default="690"/>	
	<arg name="angular" default="0.006"/>	
	<arg name="turn_v" default="0.03"/>		
	<!--  -->
	<node pkg="planning_and_controlling" type="scan_control_node" name="scan_control_node" output="screen">
                <param name="target_distance" value="$(arg target_distance)"/>
                <param name="threshold_distance" value="$(arg threshold_distance)"/> 
                <param name="Kp" value="$(arg Kp)"/>
                <param name="Ti" value="$(arg Ti)"/>
                <param name="Td" value="$(arg Td)"/>
                <param name="line_v" value="$(arg line_v)"/>
                <param name="angular_z" value="$(arg angular_z)"/>
                <param name="num_f" value="$(arg num_f)"/>    
                <param name="num_l" value="$(arg num_l)"/>
                <param name="num_r" value="$(arg num_r)"/>
                <param name="num_rf" value="$(arg num_rf)"/>
                <param name="num_rh" value="$(arg num_rh)"/>
                <param name="angular" value="$(arg angular)"/>
                <param name="turn_v" value="$(arg turn_v)"/>                    
        </node>
</launch>
